Makefile

```diff
@@ -43,9 +43,9 @@ build:
 	@go build ${GO_CMD_FLAGS} -o ${CONTROLLER_BOOTSTRAP} ./cmd/controller-bootstrap/main.go
 
 generate: build
-	@${CONTROLLER_BOOTSTRAP} generate --aws-service-alias ${AWS_SERVICE} --ack-runtime-version ${ACK_RUNTIME_VERSION} \
-    --aws-sdk-go-version ${AWS_SDK_GO_VERSION} --dry-run=${DRY_RUN} --output-path ${ROOT_DIR}/../${AWS_SERVICE}-controller \
-    --model-name ${SERVICE_MODEL_NAME} --refresh-cache=${REFRESH_CACHE} --test-infra-commit-sha ${TEST_INFRA_COMMIT_SHA}
+	@${CONTROLLER_BOOTSTRAP} generate ${AWS_SDK_GO_VERSION} ${ACK_RUNTIME_VERSION} ${TEST_INFRA_COMMIT_SHA} \
```

Comment:
nit: Because value for these arguments are very similar, I liked the earlier version where the users were able to understand using *flag name* that the semver value belonged to which repo. 

Adding as *nit* because this CLI will mostly be used with Makefile that will hide these arguments from user.

Ex: 
Earlier:
```
controller-bootstrap generate \
--aws-sdk-go-version v1.44.42 \
--ack-rutime-version v0.19.2 \
--test-infra-sha v-test-sha
```

Now
```
controller-bootstrap generate v1.44.42 v0.19.2 v-test-sha
```
Now there is a scope to shuffle the order of arguments by mistake. Curious about your take @jaypipes .

Reply:
@vijtrip2 agreed. I think there are generally two approaches taken for that:

1) use some sort of delimited argument list, like this:

```
controller-bootstrap generate s3 sdk-go:v1.44.42:ack-rt:v0.19.2:test-infra:some-sha
```

or:

2) Make these actually optional arguments by defaulting them to some discoverable value.

For example, we could make the ACK runtime version default to the latest git tag on the main branch. We might hard-code the AWS SDK Go version or take a similar lookup-git-tag approach. Same for test-infra.

And then the command line would just be:

```
controller-bootstrap generate s3
```

and if the user wanted to override the default values for various flags, they could do so:

```
controller-bootstrap generate s3 --aws-sdk-go-version v1.44.65
```

I prefer the second approach.

Reply:
I agree. Ordered arguments make sense for hierarchical command line options (like `kind delete cluster <cluster name>`) or for supplying arbitrarily ordered data (like a list of service names). Neither of these scenarios apply here

Reply:
> if the user wanted to override the default values for various flags, they could do so

Yes, we wanted to do that but the code implementation was so much simpler when making these three arguments required. And since we were using Makefile, the Makefile shell was retrieving the default values and passing them as the required arguments.